Classes/FlowBootstrapTrait.php

```diff
@@ -50,13 +54,22 @@ final protected static function bootstrapFlow(): Bootstrap
         if (!defined('BEHAT_ERROR_REPORTING')) {
             define('BEHAT_ERROR_REPORTING', E_ALL);
         }
-        self::$bootstrap = new Bootstrap('Testing/Behat');
-        Scripts::initializeClassLoader(self::$bootstrap);
-        Scripts::initializeSignalSlot(self::$bootstrap);
-        Scripts::initializePackageManagement(self::$bootstrap);
-        self::$bootstrap->setActiveRequestHandler(new FunctionalTestRequestHandler(self::$bootstrap));
-        self::$bootstrap->buildRuntimeSequence()->invoke(self::$bootstrap);
-        return self::$bootstrap;
+
+        // Boot flow with an active request handler
+        $flowBootstrap = new Bootstrap('Testing/Behat');
+        $requestHandler = new RuntimeSequenceHttpRequestHandler($flowBootstrap);
+
+        $flowBootstrap->registerRequestHandler($requestHandler);
+        $flowBootstrap->setPreselectedRequestHandlerClassName($requestHandler::class);
+
+        $serverRequestFactory = new ServerRequestFactory(new UriFactory());
+        $request = $serverRequestFactory->createServerRequest('GET', 'http://localhost');
+
+        $requestHandler->setHttpRequest($request);
+
+        $flowBootstrap->run();
```

Comment:
booting flow like this will use more steps than the beforehand used. Previously we only used these 3 (which are now part of `Bootstrap::run`

```php
Scripts::initializeClassLoader
Scripts::initializeSignalSlot
Scripts::initializePackageManagement
```

But in the `RuntimeSequenceHttpRequestHandler` we now actually boot flow fully

```php
$sequence = $this->bootstrap->buildRuntimeSequence();
$sequence->invoke($this->bootstrap);
```

Reply:
edit we previously also booted flow fully see

```
self::$bootstrap->buildRuntimeSequence()->invoke(self::$bootstrap);
```

so nothing really changed